src/main/java/teammates/storage/api/FeedbackResponseCommentsDb.java

```diff
@@ -306,7 +316,12 @@ public FeedbackResponseCommentAttributes updateFeedbackResponseComment(FeedbackR
             throw new EntityDoesNotExistException(ERROR_UPDATE_NON_EXISTENT + newAttributes.toString());
         }
         
-        frc.setGiverEmail(newAttributes.giverEmail);
+        if (isNewComment) {
+            frc.setGiverEmail(newAttributes.giverEmail);     
+        } else {
+            frc.setLastEditorEmail(newAttributes.giverEmail);
```

Comment:
those create...comment with update...comment are special cases:
1. for testing only
2. for a rare case that, the comment has already been created